201-vnet-to-vnet-bgp/azuredeploy.json

```diff
@@ -0,0 +1,239 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sharedKey": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The shared key used to establish connection between the two vNet Gateways."
+            }
+        }
+    },
+    "variables": {
+        "vNet1": {
+            "name": "[concat('vNet1-', replace(resourceGroup().location, '', ''))]",
+            "addressSpacePrefix": "10.0.0.0/23",
+            "subnetName": "subnet1",
+            "subnetPrefix": "10.0.0.0/24",
+            "gatewayName": "vNet1-Gateway",
+            "gatewaySubnetPrefix": "10.0.0.224/27",
+            "gatewayPublicIPName": "GEN-UNIQUE",
```

Comment:
GEN-UNIQUE only works in the parameters file since our CI replaces it there and then deploys it. It doesn't work in a template as it's not an ARM concept.